text/0000-include-clang-in-llvm-tools.md

```diff
@@ -0,0 +1,61 @@
+- Feature Name: Include Clang in llvm-tools
+- Start Date: 2025-08-04
+- RFC PR: [rust-lang/rfcs#3847]()
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+Include a version of `clang` and `clang++` compiled against Rust LLVM in the `llvm-tools` component in nightly.
```

Comment:
Is "on nightly" intended to be load bearing here?

In general this RFC I think needs to more clearly document the user expectations around breakage and whether we're ok making it common to reference this tool. Today, we mostly rely on slow moving distros I think to mitigate the less stable interfaces clang (and in general C / C++ compilers expose, to my knowledge), but if we're shipping it ourselves that pushes towards Rust being the one blamed for breakage.